library/src/main/java/com/google/maps/android/data/Renderer.java

```diff
@@ -553,7 +553,7 @@ private void putMarkerImagesCache(String url, String scale, BitmapDescriptor bit
      * @param url    image URL
      * @param bitmap image bitmap
      */
-    protected void cacheBitmap(String url, Bitmap bitmap) {
+    public void cacheBitmap(String url, Bitmap bitmap) {
```

Comment:
Please revert—changing the scope of `cacheBitmap` and `checkClearBitmapCache` are implementation leaks. I'll comment on `KmlLayer` on another suggested approach.

Reply:
Done https://github.com/googlemaps/android-maps-utils/pull/625/commits/482c748605b3d9a46a88118b1a9304d3bf77644d